includes/class-wgpb-products-controller.php

```diff
@@ -230,10 +230,13 @@ public function prepare_item_for_response( $post, $request ) {
 	protected function prepare_objects_query( $request ) {
 		$args = parent::prepare_objects_query( $request );
 
-		$orderby       = $request->get_param( 'orderby' );
-		$order         = $request->get_param( 'order' );
-		$cat_operator  = $request->get_param( 'cat_operator' );
-		$attr_operator = $request->get_param( 'attr_operator' );
+		$orderby            = $request->get_param( 'orderby' );
+		$order              = $request->get_param( 'order' );
+		$cat_operator       = $request->get_param( 'cat_operator' );
+		$attr_operator      = $request->get_param( 'attr_operator' );
+		$attributes         = $request->get_param( 'attributes' );
+		$tax_relation       = $request->get_param( 'tax_relation' );
```

Comment:
Looks like there was a rebase issue here, `$attributes` & `$tax_relation` were removed in #401

Reply:
Thanks, I did this merge and must have just missed it.